flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/AggregateFunction.scala

```diff
@@ -35,6 +36,18 @@ abstract class AggregateFunction[T] extends UserDefinedFunction {
   def createAccumulator(): Accumulator
 
   /**
+    * Retract the input values from the accumulator instance.
```

Comment:
I was thinking further for the dataStream retraction where the source table could send out of date retraction message. But let us add this assumption for now.